nss-day-cohort-29/welcome-to-nashville-knights
# Sg welcome to nashville knights

Description

 fetch api park



# Testing Instructions for Change Made
git checkout sg-welcome-to-nashville-knights
check to see if fetch call is working fine
# Checklist:
